sparse/lib/systemd/system/wlan-module-load.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Load wifi module
+After=local-fs.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=/bin/sleep 5
+ExecStart=/sbin/modprobe bcmdhd firmware_path=/system/vendor/firmware/brcm/fw_bcmdhd_4356a2_pcie.bin nvram_path=/system/vendor/firmware/brcm/nvram_pcie_4356_a2.cal
```

Comment:
Wouldn't be better loading up the bcmdhd module via the facility systemd provides (systemd-modules-load, /etc/modules-load.d)? We could get rid of this unit and eventually we'll need to only adjust the startup order. 

What do you think?

Reply:
The reason for the sleep 5 hack is.to wait for the droid filesystems to be mounted, including the one which contains the wlan mac addr.  options are to fix the load order before loading the module,  or, store the mac addr on another filesystem. Thoughts?

Reply:
Hmm... `After=local-fs.target` should be enough I think (system.mount which mounts /system gets installed in the local-fs target). Perhaps also adding `Requires=local-fs.target` would enforce that even more...

Reply:
Mac addr is on /config/wifi/mac.txt btw